code42_connector.py

```diff
@@ -327,6 +327,13 @@ def _handle_reactivate_user(self, param, action_result):
             phantom.APP_SUCCESS, f"{username} was reactivated"
         )
 
+    @action_handler_for("get_user_profile")
+    def _handle_get_user_profile(self, param, action_result):
+        username = param["username"]
+        profile = self._client.detectionlists.get_user(username)
```

Comment:
If we are in a playbook that calls `create_user` and then after that calls this action, it will fail because the `get_user()` method fails for newly created users for a long time after.

If we use the `get_by_id()` method, it would work better in this case, but that would require having to first get figure out the ID of the user via `user_id = self._get_user_id(username)`

I think it would be worth it though. There are some use-cases involving these two steps